content/pages/education/apply.md

```diff
@@ -95,3 +113,29 @@ You must apply for education benefits using eBenefits if you're:
 </div>
 
 <div markdown="0"><br></div>
+
+<script type="text/javascript">
+  // I'm open to suggestions on how to not do this here
+
+  function toggleClass(elementId, className) {
+    document.getElementById(elementId).classList.toggle(className);
+  }
+
+  // Toggle the expandable apply fields
+  document.getElementById('apply-expander-button')
+    .addEventListener('click', function () {
+      toggleClass('apply-expander-content', 'expander-content-closed');
+      toggleClass('apply-expander-button', 'va-button-primary');
+    });
+
+  // Make the go button go to the right place
+  document.getElementById('apply-go-button')
+    .addEventListener('click', function () {
+      var selectedForm = document.querySelector('input[name="form-selection"]:checked');
+
+      console.log('selectedForm:', selectedForm);
+      if (selectedForm) {
+        location.assign('/education/apply-for-education-benefits/application/' + selectedForm.value + '/introduction');
```

Comment:
This seems pretty contrived, so I'm open to suggestions here too. I did this because it was short, sweet, and to the point.